src/test/java/cms/sre/emailnotifier/service/EmailServiceTest.java

```diff
@@ -18,15 +20,20 @@
 @SpringBootTest(classes = App.class)
 public class EmailServiceTest{
```

Comment:
You should test to make sure an instance of EmailService is autowired (even though you aren't testing that instance specificially).